dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasics/TransferBasics.cs

```diff
@@ -5,21 +5,35 @@
 
 // This Amazon S3 client uses the default user credentials
 // defined for this computer.
+using Microsoft.Extensions.Configuration;
+
 IAmazonS3 client = new AmazonS3Client();
 var transferUtil = new TransferUtility(client);
-
-// Change the following values to an Amazon S3 bucket that
-// exists in your AWS account.
-var bucketName = "doc-example-bucket1";
+IConfiguration _configuration;
+
+_configuration = new ConfigurationBuilder()
+    .SetBasePath(Directory.GetCurrentDirectory())
+    .AddJsonFile("settings.json") // Load test settings from JSON file.
+    .AddJsonFile("settings.local.json",
+        true) // Optionally load local settings.
+    .Build();
+
+// Edit the values in settings.json to use a bucket and files that
+// exist on your AWS account and on the local computer where you
+// run this scenario.
+var bucketName = _configuration["BucketName"];
 var localPath = $"{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}\\TransferFolder";
 
 DisplayInstructions();
 
 PressEnter();
 
+Console.WriteLine();
+
 // Upload a single file to an S3 bucket.
-var fileToUpload = "UploadTest.docx";
+DisplayTitle("Upload a single file");
```

Comment:
Missing punctuation?

Reply:
If this text functions as a title, no ending punctuation is required.